nipype/info.py

```diff
@@ -135,47 +135,47 @@ def get_nipype_gitversion():
 VERSION = __version__
 PROVIDES = ['nipype']
 REQUIRES = [
-    'nibabel>=%s' % NIBABEL_MIN_VERSION,
-    'networkx>=%s,<=%s ; python_version < "3.0"' % (NETWORKX_MIN_VERSION, NETWORKX_MAX_VERSION_27),
+    'click>=%s' % CLICK_MIN_VERSION,
+    'configparser; python_version <= "3.4"',
+    'funcsigs',
+    'future>=%s' % FUTURE_MIN_VERSION,
+    'futures; python_version == "2.7"',
     'networkx>=%s ; python_version >= "3.0"' % NETWORKX_MIN_VERSION,
-    'numpy>=%s,!=%s ; python_version == "2.7"' % (NUMPY_MIN_VERSION, NUMPY_BAD_VERSION_27),
+    'networkx>=%s,<=%s ; python_version < "3.0"' % (NETWORKX_MIN_VERSION, NETWORKX_MAX_VERSION_27),
+    'neurdflib',
```

Comment:
this one has to come at the end. we are using pip install order hack to install a same named library but with additions that remain to be merged into a release.